example/90-shoot.yaml

```diff
@@ -314,10 +314,10 @@ spec:
     # When the shoot shall use a cluster domain no domain and no providers need to be provided - Gardener will
     # automatically compute a correct domain based on the default domains in the garden cluster.
     domain: crazy-botany.core.my-custom-domain.com
+    # Provider configuration required if custom shoot domain is configured.
   # providers:
   # - type: aws-route53
   #   secretName: my-custom-domain-secret
-  #   primary: true # `true` indicates that this provider is also used to manage the shoot domain `.spec.dns.domain`
```

Comment:
Is it not too early to remove the primary field? You still need to set it.

Reply:
It is set/defaulted by Gardener if not set. The removal from the example was more or less a simplification and encouragement to not set it.